runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/config/StreamingEngineComputationConfigFetcher.java

```diff
@@ -259,7 +272,7 @@ public void stop() {
   @SuppressWarnings("FutureReturnValueIgnored")
   private void schedulePeriodicGlobalConfigRequests() {
     globalConfigRefresher.scheduleWithFixedDelay(
-        () -> fetchGlobalConfig().ifPresent(onStreamingConfig),
+        () -> fetchGlobalConfig().ifPresent(configManager::setConfig),
```

Comment:
since we don't use other methods on the configManager, the Consumer seems better for testing.

Reply:
Added a StreamingEnginePipelineConfigManager interface.